package.json

```diff
@@ -22,9 +22,9 @@
     "@angular/platform-browser-dynamic": "^4.3.0",
     "@angular/router": "^4.3.0",
     "@webcomponents/custom-elements": "1.0.0",
-    "clarity-angular": "~0.10.10",
-    "clarity-icons": "~0.10.10",
-    "clarity-ui": "~0.10.10",
+    "clarity-angular": "~0.10.12",
```

Comment:
Why not just do `^0.10.0` so that we dont have to update this every week?

Reply:
Hmm, I think there was a reason but can't remember what it was and just have a note to remember to update it. I'll change it to that.